spec/factories/orders.rb

```diff
@@ -1,12 +1,22 @@
 FactoryGirl.define do
   factory :order do
-    name "Bob Schmob"
-    description "Bobs Burgers"
-    price 1
-    order_date "2017-09-09 10:15:58"
-    delivery_date "2017-09-09 10:15:58"
-    address "Bobs address"
-    phone "070444444"
-    allergies "peanuts"
+    allergies 'peanuts'
+    delivery_method 'pick up'
```

Comment:
Is the delivery_method "delivery" missing? Cannot find it

Reply:
No, this you can't have both at the same time.

Reply:
right

Reply:
one order can only have pickup or delivery. Not both.